spec/features/user_creates_account_spec.rb

```diff
@@ -2,25 +2,25 @@
 
 feature "user can register for an account" do
 
+  let(:user) { FactoryGirl.create(:user) }
+
   scenario "user succesfully creates a new account" do
-    user = FactoryGirl.build(:user)
+    new_user = FactoryGirl.build(:user)
     visit new_user_registration_path
-    fill_in 'Username', with: user.username
-    fill_in 'Email', with: user.email
-    fill_in 'Password', with: user.password
-    fill_in 'Password confirmation', with: user.password
+    fill_in 'Username', with: new_user.username
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.